tests/baselines/reference/api/typescript.d.ts

```diff
@@ -4448,6 +4449,17 @@ declare namespace ts {
     function createAbstractBuilder(rootNames: ReadonlyArray<string> | undefined, options: CompilerOptions | undefined, host?: CompilerHost, oldProgram?: BuilderProgram, configFileParsingDiagnostics?: ReadonlyArray<Diagnostic>, projectReferences?: ReadonlyArray<ProjectReference>): BuilderProgram;
 }
 declare namespace ts {
+    function readBuilderProgram(compilerOptions: CompilerOptions, readFile: (path: string) => string | undefined): (EmitAndSemanticDiagnosticsBuilderProgram & SemanticDiagnosticsBuilderProgram) | undefined;
```

Comment:
`EmitAndSemanticDiagnosticsBuilderProgram` extends `SemanticDiagnosticsBuilderProgram`, so returning an intersection of the two seems redundant

Reply:
Thanks for finding it. That refactoring was done later so missed this.